### DISCUSSION TITLE
Unable to build from source code in Visual Studio

### DISCUSSION THREAD

**Discussion by sryze** [microsoft/terminal#15188] [General]
![image](https://user-images.githubusercontent.com/4525736/232254977-c7ed27e6-1290-43ed-865c-f8273661f5df.png)

It seems that the solution configuration is messed up. When I try to run the CascadiaPackage project (as mentioned [here](https://github.com/microsoft/terminal/blob/main/doc/building.md)) it produces an error:

> The project needs to be deployed before we can debug.

Once I check the Deploy and Build checkbox next to CascadiaPackage in Configuration Manager, I get another error:

> Specified EntryPointExe 'WindowsTerminal.exe' was not found in the project outputs.

And so on.

Am I missing something? I just want to run Windows Terminal under a debugger (to investigate a bug).

Thanks.

**Comment by zadjii-msft** [comment]
I'm pretty sure that's just cause VS defaults to "AnyCPU", when it needs to be x64 (or x86). See also #14634

**Comment by sryze** [comment]
I was able to build & run it after checking "Build" next to each project (except tests) and changing the active configuration from x86 to x64. This didn't get rid of the popup notification though, I guess it can be ignored.
